docs/software/comanage-troubleshooting-guide.md

```diff
@@ -0,0 +1,77 @@
+COmanage Troubleshooting Guide
+====
+Introduction
+----
+The purpose of this guide is to be a resource for COmanage administrators to find solutions to commonly encountered problems.
+
+
+COManage Troubleshooting Items
+----
+This section contains some of the issues you may encounter when interacting with the OSG COmanage.
+
+Problem list:
+
+ - [Tiger-Dex Unable to match User to COmanage CO Person identity](#tiger-dex-unable-to-match-user-to-comanage-co-person-identity)
+
+###Tiger-Dex Unable to match User to COmanage CO Person identity
+
+Under certain circumstances Dex may not be able to match a user to their COmanage identity,
+ possibly due to incorrect and/or missing organization identities/identifiers.
+
+####Symptoms
+
+Symptoms include:
+
+ - Tiger-Dex credentials not including some or all of a users group permissions.
+ - Not being able to connect to the Tiger cluster.
+
+####Next actions
+
+- Ensure all of the user's org identities / identifiers required by LDAP exist and match across COmanage and LDAP.
+    - Found at `https://registry.cilogon.org/registry/ldap_provisioner/co_ldap_provisioner_targets/edit/6`
+- Setting the missing/mismatched value manually did not seem to work, 
```

Comment:
This should be an enumerated list and this second point should be that the user should log in.

From https://osg-htc.org/technology/documentation/writing-documentation/#tips-for-writing-procedural-instructions

> Number all steps (as opposed to using bullets)